test/verify/check-multi-os

```diff
@@ -50,7 +50,7 @@ class TestMultiOS(MachineCase):
     }
 
     def check_spawn(self, b, address):
-        # Firefox cannot do `cockpit.spawn` as it returns promise
+        # HACK: Firefox cannot do `cockpit.spawn` as it returns promise
```

Comment:
This message does not seem to be fully correct.
Firefox did not support promises, but it seems it does now (\o/). So this one technically should work. I'll take a look if we can really drop it in different PR.